### PR TITLE
Fix typo on .complement() naming

### PR DESCRIPTION
**Pull request by calvintam236**
See [original name](https://github.com/bgrins/TinyColor#complement).

### CODE REVIEW

**Review by WieFel — APPROVED** [review]

**Comment by calvintam236** [comment]
@WieFel can you approve again after merging the changelog conflict?

**Review by WieFel — APPROVED** [review]

**Pull request merged**
